common/djangoapps/util/views.py

```diff
@@ -308,7 +322,7 @@ def _record_feedback_in_zendesk(
 
 
 def _record_feedback_in_datadog(tags):
-    datadog_tags = [u"{k}:{v}".format(k=k, v=v) for k, v in tags.items()]
+    datadog_tags = [u"{k}:{v}".format(k=k, v=v) for k, v in sorted(tags.items())]
```

Comment:
That doesn't work because we're not directly comparing the tags in the test, we're using mocks to verify that a certain method was called with specific arguments. See https://github.com/edx/edx-platform/pull/14236/files#diff-8f395611f456cfd137e4252981eedf31R139